Sources/Build/BuildPlan/BuildPlan+Swift.swift

```diff
@@ -43,7 +43,7 @@ extension BuildPlan {
                     let libraries = try self.parseXCFramework(for: target, triple: swiftTarget.buildParameters.triple)
                     for library in libraries {
                         library.headersPaths.forEach {
-                            swiftTarget.additionalFlags += ["-I", $0.pathString, "-Xcc", "-I", "-Xcc", $0.pathString]
+                            swiftTarget.libraryBinaryPaths.insert($0)
```

Comment:
https://github.com/swiftlang/swift-package-manager/pull/7998#issuecomment-2443480388

Do you mean Line 46 or Line 48 here? @xedin 

Considering the `po` result, I guess you mean Line 46. But it would conflict with your previous comment here.

> To that end, I think we should remove the code that adds -I and -Xcc -I from BuildPlan+Swift.swift and replace it with
> 
> library.headersPaths.forEach {
>  swiftTarget.libraryBinaryPaths.insert($0)
> }

Reply:
I mean `swiftTarget.libraryBinaryPaths.insert(library.libraryPath)` on line 48, `libraryPath` doesn't actually point to the place that `-F` would recognize anyway.

Reply:
This is weird. I do not know the what "-F" is recognizing. But according to the test case, deleting Line 46 is fine while deleting Line 48 would make the test case fail with "No such module FooKit".

Saying that, I do think it is valuable we add real xcframework test case (via a trusted local one or fix the CI to build it on the fly). The simple flags check may be  doubtful here.

There is other reviewer have a +1 for the real test case.

https://github.com/swiftlang/swift-package-manager/pull/7998/files?w=1#r1822483995